docs/tempo/website/configuration/tls.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Configure TLS Communication
+weight: 70
```

Comment:
I made up some numbers.

Reply:
The numbers determine the order the pages appear in the left-hand navigation.

Reply:
I'll take your suggestion on where to nest it.

Reply:
Here's the current left nav under Configuration and their assigned weights. I am not sure that these topics are grouped in any particular order. The first three topics deal with permissions, so maybe under Azure blog storage? Is there another topic that TLS is more related to that would make more sense? 

GCS - weight 10 
S3 - weight 20
Azure - weight 30
Query - weight 30
Compression - weight 50
Multi-tenancy - weight 60
Manifest - weight 70
Parquet - weight 75
Polling - weight 80 

With a weight of 70, the TLS page would appear either above or below Manifest. If you want the page to appear under Azure blog storage, then maybe a weight of 35 (and change the weight of Query to 40).

![image](https://user-images.githubusercontent.com/104772500/200987260-7e2bc358-cac7-400a-9676-64bb54d02dc6.png)